phpBB/includes/functions_acp.php

```diff
@@ -294,34 +320,125 @@ function build_cfg_template($tpl_type, $key, &$new_ary, $config_key, $vars)
 			$min = isset($tpl_type[1]) ? (int) $tpl_type[1] : false;
 			$max = isset($tpl_type[2]) ? (int) $tpl_type[2] : false;
 
-			$tpl = '<input id="' . $key . '" type="' . $tpl_type[0] . '"' . (( $min !== false ) ? ' min="' . $min . '"' : '') . (( $max !== false ) ? ' max="' . $max . '"' : '') . ' name="' . $name . '" value="' . $new_ary[$config_key] . '" />';
+			$tpl = [
+				'tag'		=> 'input',
+				'id'		=> $key,
+				'type'		=> $tpl_type[0],
+				'name'		=> $name,
+				'min'		=> $min !== false ? $min : '',
```

Comment:
```$min ?: ''```

shorter version, same for max

Reply:
The check is actually stricter than what you proposed and fully mirrors the previous check.